oneview_redfish_toolkit/blueprints/resource_block.py

```diff
@@ -149,3 +141,98 @@ def get_resource_block_computer_system(uuid, serial):
         # In case of error log exception and abort
         logging.exception('Unexpected error: {}'.format(e))
         abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@resource_block.route(
+    ResourceBlock.BASE_URI + "/<uuid>/EthernetInterfaces/<id>",
+    methods=["GET"])
+def get_resource_block_ethernet_interface(uuid, id):
+    """Get the Redfish ResourceBlock of type Network for the given UUID and ID.
+
+        Return ResourceBlock redfish JSON for a given UUID and ID.
+        Logs exception of any error and return Internal Server
+        Error or Not Found.
+
+        Returns:
+            JSON: Redfish json with ResourceBlock.
+    """
+
+    try:
+        server_profile_template = \
+            g.oneview_client.server_profile_templates.get(uuid)
+
+        connSettings = server_profile_template["connectionSettings"]
+        connection = None
+
+        for conn in connSettings["connections"]:
+            if str(conn["id"]) == id:
+                connection = conn
+                break
+
+        if not connection:
+            raise OneViewRedfishError("Ethernet interface not found")
+
+        network = g.oneview_client.ethernet_networks.get(conn["networkUri"])
+
+        ethernet_interface = ResourceBlockEthernetInteface(
+            server_profile_template, connection, network)
+
+        json_str = ethernet_interface.serialize()
+
+        # Build response and returns
+        response = Response(
+            response=json_str,
+            status=status.HTTP_200_OK,
+            mimetype="application/json")
+        response.headers.add("ETag", "W/" + server_profile_template["eTag"])
+
+        return response
+    except HPOneViewException as e:
+        # In case of error log exception and abort
+        logging.exception(e)
+        abort(status.HTTP_404_NOT_FOUND, "Resource block not found")
+
+    except OneViewRedfishError as e:
+        # In case of error log exception and abort
+        logging.exception('Unexpected error: {}'.format(e))
+        abort(status.HTTP_404_NOT_FOUND, e.msg)
+
+    except Exception as e:
```

Comment:
This code could be placed on the app.py as a new Exception handler for the flask app, if it already doesn't do it